monai/transforms/intensity/array.py

```diff
@@ -2293,6 +2293,7 @@ def __call__(self, image: NdarrayOrTensor):
 class ComputeHoVerMaps(Transform):
     """Compute horizontal and vertical maps from an instance mask
     It generates normalized horizontal and vertical distances to the center of mass of each region.
+    Input data with the size of [CxHxW[xD]], which channel dim will temporarily removed for calculating coordinates.
```

Comment:
Input data here has to be 1xHxW[xD] since squeeze will not remove non-singleton dimensions.

Reply:
Have updated it, thanks!